wgpu-hal/src/gles/egl.rs

```diff
@@ -1092,6 +1092,7 @@ impl Surface {
             .map_err(|e| {
                 log::error!("swap_buffers failed: {}", e);
                 crate::SurfaceError::Lost
+                // TODO: should we unset the current context here?
```

Comment:
I suspect this should be handed by some unset-on-drop mechanism? Don't have to fix this in this PR though.